docs/docs/tutorials/gettingstarted.md

```diff
@@ -0,0 +1,197 @@
+- Once the build finishes, the initial SpringBootApplication run will have failed. Now re-run it with your newly saved environment variables
+- You should now be able to go to localhost:8080 and see the project running successfully.
+
+## Getting it to Run (Long Version)
+Now we have pulled down the Springboard project, let's open it in your favorite IDE. I'm going to be using IntelliJ.
+
+- Open IntelliJ
+- File -> Open -> rvroger
+
+This is going to import the "rvroger" Gradle project. It can take a few minutes to pull in all the dependencies.
+ It's tempting to click "Background" on the task and start changing things, but go make a cup of tea or coffee and 
+ just let it finish. (It took 2 minutes and 19 seconds for my computer to finish getting everything setup)
+
+Now let's run the application and see what we have. If you open "src/main/kotlin/org.springboard" and right click
+ SpringboardApplication.kt, the "Run" option should be available in the menu (or if you are a shortcut wizard -
+  "CTRL + SHIFT + F10"). Since this is the first time you will have built the project, it is going to take a bit of 
+  time to run. The "buildAssets" step in particular will take up the majority of this time. Mine took another 2 
+  minutes and 46 seconds to finish the build.
+
+Now IntelliJ has finished the build- it's going to start running our application!
+
+Anddd we get a horrible error blowup:
+
+[![Error Logs for Initially Starting IntelliJ Without DEV Spring Profile](images/getting-started-missing-dev.png)](images/getting-started-missing-dev.png)
+
+If we drill down and read the stack trace- we see the root cause here: 
+
+```
+Driver org.postgresql.Driver claims to not accept jdbcUrl, jdbc:postgresql://${DB_SERVER}:${DB_PORT}/${DB_NAME}
+```
+
+Take a look at our main resources- we have application.properties and application-dev.properties. By default, Spring
+ is going to pick up application.properties, which, if we open it, expects to find environment variables for the 
+ database configuration. 
+ 
+ [![Initial Application Properties File](images/getting-started-application-properties.png)](images/getting-started-application-properties.png)
+
+We are just trying to run a development instance here- so looking at application-dev.properties, it's already 
+configured some test database properties. 
+ 
+ [![Initial DEV Application Properties File](images/getting-started-dev-application-properties.png)](images/getting-started-dev-application-properties.png)
+ 
+Since the DEV application properties file contains everything we need to run a local development instance of our 
+application, we need to make sure that this is the file being picked up when we run. Let's edit the Run Configuration 
+for SpringboardApplication so that it uses the DEV spring profile. Since we tried  to run this already- 
+IntelliJ has a Run Configuration setup for us if we "Edit Configurations" from the run configuration menu.
+  
+We need to add an environment variable:
```

Comment:
You can also configure the profile directly in intellij.

Reply:
How?

Reply:
Scroll down in the Run Configuration window for a Spring run and you'll see it.